go/vt/vttablet/tabletserver/vstreamer/planbuilder.go

```diff
@@ -257,7 +258,7 @@ func getKeyspaceID(values []sqltypes.Value, vindex vindexes.Vindex, vindexColumn
 	for _, col := range vindexColumns {
 		vindexValues = append(vindexValues, values[col])
 	}
-	destinations, err := vindexes.Map(vindex, nil, [][]sqltypes.Value{vindexValues})
+	destinations, err := vindexes.Map(context.TODO(), vindex, nil, [][]sqltypes.Value{vindexValues})
```

Comment:
Is this context okay?

Reply:
here the vcursor is nil which means context also does have any meaning here. 
these are functional vindex map call which does not use context.

Reply:
if this is the case, we should use `context.Background()` instead. 
My understanding of `context.TODO()` is that is should be use much as a ``// TODO`` - this is not correct, and should be fixed.